tools/backups2datalad.py

```diff
@@ -214,13 +227,28 @@ def get_annex_hash(filepath):
             local_assets = set(dataset_files(dsdir))
             local_assets.discard(dsdir / dandiset_metadata_file)
             asset_metadata = []
+            saved_metadata = {}
+            try:
+                with (dsdir / ".dandi" / "assets.json").open() as fp:
+                    for md in json.load(fp):
+                        saved_metadata[md["path"]] = md
+            except FileNotFoundError:
+                pass
             for a in assets:
                 dest = dsdir / a["path"].lstrip("/")
                 deststr = str(dest.relative_to(dsdir))
                 local_assets.discard(dest)
                 am = deepcopy(a)
                 am["modified"] = str(am["modified"])
                 asset_metadata.append(am)
+                if (
+                    self.force is None or "check" not in self.force
+                ) and am == saved_metadata.get(am["path"]):
+                    log.info(
```

Comment:
I will lower this one to `.debug` -- in a typical run it just fills up the screen, hard to monitor for "progress"

Reply:
Done.